test/e2e/core.go

```diff
@@ -37,55 +37,61 @@ func CoreDump(dir string) {
 	}
 	provider := testContext.Provider
 
-	// requires ssh
-	if !providerIs(providersWithSSH...) {
+	// requires ssh to master machine
+	if !providerIs(providersWithMasterSSH...) {
 		fmt.Printf("Skipping SSH core dump, which is not implemented for %s", provider)
 		return
 	}
 
-	// Get all nodes' external IPs.
-	hosts, err := NodeSSHHosts(c)
+	// I wish there was a better way to get the master IP...
```

Comment:
FYI - this is just changing the order. We first copy files from master machine and only then from nodes (it was the oposite order before).